packages/plugins/graphql-jit/src/index.ts

```diff
@@ -48,10 +44,7 @@ export const useGraphQlJit = (
     cache?: JITCache;
   } = {},
 ): Plugin => {
-  const jitCacheByDocumentString =
-    typeof pluginOptions.cache !== 'undefined'
-      ? pluginOptions.cache
-      : new LRUCache<string, JITCacheEntry>({ max: DEFAULT_MAX, ttl: DEFAULT_TTL });
+  const jitCacheByDocumentString = pluginOptions.cache;
```

Comment:
This means that the cache is now disabled by default ?

Reply:
Yes